Add Helm Chart repositories add action on Add page

vikram-raj opened the pull request:
**Story:** https://issues.redhat.com/browse/ODC-6635

**Description:** 
Add the Project Helm Chart repositories add card on the Aad page

**Screen shots:**
![image](https://user-images.githubusercontent.com/2561818/164623872-18292529-ebcf-4386-b3e5-e5efcbb11e27.png)

![Kapture 2022-04-22 at 12 36 15](https://user-images.githubusercontent.com/2561818/164623748-c0e1fb4e-1d35-489b-a8d4-f527273fe6c4.gif)

vikram-raj commented:
/kind feature

/cc @debsmita1

debsmita1 commented:
The `Cancel` button on the form doesn't work

vikram-raj commented:
> The `Cancel` button on the form doesn't work

@debsmita1 should it will take back to Add page?

karthikjeeyar commented on frontend/packages/helm-plugin/console-extensions.json:
```diff
@@ -119,6 +127,20 @@
+      "groupId": "helm-chart-repositories",
+      "href": "/ns/:namespace/projecthelmchartrepositories/~new",
+      "label": "%helm-plugin~Helm Chart repositories%",
+      "description": "%helm-plugin~Add a Helm Chart Respository in the namespace%",
```
shouldn't this be ` Helm Chart repository` ? We use lowercase in the project helm chart form's helptext. cc: @debsmita1 
 
```suggestion
      "description": "%helm-plugin~Add a Helm Chart repository in the namespace%",
```

debsmita1 commented on frontend/packages/helm-plugin/console-extensions.json:
```diff
@@ -105,6 +105,14 @@
+    "type": "dev-console.add/action-group",
+    "properties": {
+      "id": "helm-chart-repositories",
+      "name": "%helm-plugin~Helm Chart repositories%",
```
```suggestion
      "name": "%helm-plugin~Project Helm Chart repositories%",
```

vikram-raj replied:
Updated

debsmita1 commented:
/cc @serenamarie125

karthikjeeyar commented on frontend/packages/helm-plugin/locales/en/helm-plugin.json:
```diff
@@ -3,8 +3,10 @@
+  "Project Helm Chart repositories": "Project Helm Chart repositories",
   "Helm Chart": "Helm Chart",
   "Browse the catalog to discover and install Helm Charts": "Browse the catalog to discover and install Helm Charts",
+  "Add a Helm Chart respository in the namespace": "Add a Helm Chart respository in the namespace",
```
```suggestion
  "Add a Helm Chart repository in the namespace": "Add a Helm Chart respository in the namespace",
```

karthikjeeyar commented:
Verified on cluster, it works as expected

/lgtm

karthikjeeyar commented:
/label qe-approved

invincibleJai commented:
/approve

debsmita1 commented:
Propagating docs and px acks from the Epic
/label docs-approved
/label px-approved

kjack-redhat commented:
Looks good to me.

debsmita1 commented:
@vikram-raj The namespace dropdown is missing from the form

vikram-raj commented:
> @vikram-raj The namespace dropdown is missing from the form

There is no namespace field in the form [CreateProjectHelmChartRepositoryForm.tsx](https://github.com/openshift/console/blob/master/frontend/packages/helm-plugin/src/components/forms/ProjectHelmChartRepository/CreateProjectHelmChartRepositoryForm.tsx)

debsmita1 commented:
> > @vikram-raj The namespace dropdown is missing from the form
> 
> There is no namespace field in the form [CreateProjectHelmChartRepositoryForm.tsx](https://github.com/openshift/console/blob/master/frontend/packages/helm-plugin/src/components/forms/ProjectHelmChartRepository/CreateProjectHelmChartRepositoryForm.tsx)

Sorry, I meant the namespace bar

vikram-raj commented:
> > > @vikram-raj The namespace dropdown is missing from the form
> > 
> > 
> > There is no namespace field in the form [CreateProjectHelmChartRepositoryForm.tsx](https://github.com/openshift/console/blob/master/frontend/packages/helm-plugin/src/components/forms/ProjectHelmChartRepository/CreateProjectHelmChartRepositoryForm.tsx)
> 
> Sorry, I meant the namespace bar

Added the namespace bar and updated the PR. PTAL. Thanks

debsmita1 commented:
verified all the changes on a dev cluster. Thank you @vikram-raj 

/lgtm

rohitkrai03 reviewed (approved):
/approve
